src/test/java/com/hivemq/bootstrap/StartupListenerVerifierTest.java

```diff
@@ -15,48 +15,47 @@
  */
 package com.hivemq.bootstrap;
 
-import com.google.common.collect.Lists;
 import com.hivemq.configuration.service.entity.TcpListener;
 import com.hivemq.exceptions.UnrecoverableException;
 import org.junit.Test;
 
 import java.util.ArrayList;
+import java.util.List;
 
 /**
  * @author Dominik Obermaier
  */
 public class StartupListenerVerifierTest {
 
-
     @Test(expected = UnrecoverableException.class)
     public void test_verifier_verify_only_listener_failed() throws Exception {
-        final ListenerStartupInformation failed = ListenerStartupInformation.failedListenerStartup(2000,
-                new TcpListener(2000, "0.0.0.0"),
-                new RuntimeException("reason"));
-        new StartupListenerVerifier(Lists.newArrayList(failed)).verifyAndPrint();
+        final ListenerStartupInformation failed =
+                ListenerStartupInformation.failedListenerStartup(new TcpListener(2000, "0.0.0.0"),
+                        new RuntimeException("reason"));
+        new StartupListenerVerifier(List.of(failed)).verifyAndPrint();
```

Comment:
is there any version for a single element? we could avoid allocating a list

Reply:
you mean the constructor or the class itself?

Reply:
the constructor, like `new StartupListenerVerifier(failed)`

Reply:
in that case, yes. LGTM